demo/demo-dnt-helper.js

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
I copied the original `mozilla-dnt-helper.js` file and pasted a new one here for demo purposes. I didn't want the OG file to have the function to display things in the HTML file.